static/js/components/search/FilterVisibilityToggle_test.js

```diff
@@ -108,4 +108,53 @@ describe('FilterVisibilityToggle', () => {
     icon.simulate('click');
     assert(setFilterVisibility.called);
   });
+
+  const filterIds = {
+    "birth_location": "profile.birth_country",
+    "semester": "program.semester_enrollments.semester",
+    "education_level": "profile.education.degree_name",
+    "company_name": "profile.work_history.company_name"
+  };
+  for (const [key, value] of Object.entries(filterIds)) {
+    it(`looks up ${key} using our filter id list`, () => {
+      sandbox.stub(FilterVisibilityToggle.prototype, 'getResults').returns({
+        aggregations: {
+          [`${value}123`]: {
+            doc_count: 2
+          }
+        }
+      });
+      props.filterName = key;
+      const wrapper = renderWrappedToggle(props, <div id={key}>Test Text</div>);
+      assert.equal(wrapper.find("Icon").props().className, "");
+    });
+  }
+
+  it('looks up a key not in our filter list', () => {
```

Comment:
better description: `"shows the toggle icon the filter name is a prefix of a key in the ES results"`